hilti/runtime/src/types/bytes.cc

```diff
@@ -40,7 +40,7 @@ std::tuple<bool, Bytes::const_iterator> Bytes::find(const Bytes& v, const const_
             if ( *x++ != *y++ )
                 break;
 
-            if ( y == v.end() )
+            if ( y == v.unsafeEnd() )
```

Comment:
The variable names weren't changed here, but I spent way more time than I'm willing to admit just figuring out what this function is doing. If I read `View::_findBackward` first that would've been nice since those names are slightly more descriptive. I think it'd be good to touch that up at some point :)

Reply:
Agree, this could use some cleanup (and some comments). I'll rename the function parameters to be a more descriptive, but will leave the rest to some other time.